### PR TITLE
Added initial functionality to scrap a list of blinks

### PR DESCRIPTION
**Pull request by dhavalsavalia**
Some books, for some weird reason are not categorized in the Blinkist catalog. Hence, one need to manually scrap those books. This will help them scrapping all of them.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def check_cooldown(value):
 parser.add_argument('--concat-audio', action='store_true', default=False, help='Concatenate the audio blinks into a single file and tag it. Requires ffmpeg')
 parser.add_argument('--no-scrape', action='store_true', default=False, help='Don\'t scrape the website, only process existing json files in the dump folder')
 parser.add_argument('--book', default=False, help='Scrapes this book only, takes the blinkist url for the book (e.g. https://www.blinkist.com/en/books/... or nhttps://www.blinkist.com/en/nc/reader/...)')
+parser.add_argument('--books', default=False, help='Scrapes the list of books only, takes a txt file with the list of blinkist urls for the books (e.g. https://www.blinkist.com/en/books/... or nhttps://www.blinkist.com/en/nc/reader/...)')
```
Comment (leoncvlt) [inline main.py:23]: Typo? (**n**https)

File: README.md [hunk 2]
```diff
@@ -41,6 +41,10 @@ optional arguments:
   --book BOOK          Scrapes this book only, takes the blinkist url for the
                        book (e.g. https://www.blinkist.com/en/books/... or
                        nhttps://www.blinkist.com/en/nc/reader/...)
+  --books TEXT_FILE    Scrapes the list of books only, takes a txt file with the 
+                       list of blinkist urls for the books
+                       (e.g. https://www.blinkist.com/en/books/... 
+                       or nhttps://www.blinkist.com/en/nc/reader/...)
```
Comment (leoncvlt) [inline README.md:47]: Typo? (**n**https)

**Comment by leoncvlt** [comment]
Thanks for this! Sorry it took a while to review it. Just need to fix that little typo and good to go!

File: main.py (hunk 1 above)
Comment (dhavalsavalia) [reply main.py:23]: Oh yes! My bad. I'll fix it.

**Comment by leoncvlt** [comment]
Thanks again! 👍

**Comment by dhavalsavalia** [comment]
No problem! Stay safe. :)

**Pull request merged**